src/cmd/setup.rs

```diff
@@ -82,3 +84,18 @@ fn run_bash_completion(_args: &ArgsBashCompletion) -> Result<()> {
     );
     Ok(())
 }
+
+#[derive(FromArgs, PartialEq, Debug)]
+/// Print zsh completion instructions for lium
+#[argh(subcommand, name = "zsh-completion")]
+pub struct ArgsZshCompletion {}
+fn run_zsh_completion(_args: &ArgsZshCompletion) -> Result<()> {
+    run_bash_completion(&ArgsBashCompletion {})?;
```

Comment:
This will show the instruction for bash like  "Please run `source  ~/.bash_completion` for the current shell." too.
Can you move common code into a new `install_bash_completion()`  and call it from both `run_bash/zsh_completion()`?